lib/generators/setup_generator.rb

```diff
@@ -34,64 +37,64 @@ def setup_files
   private
 
   def copy_configs
-    puts "Copying in configuration files"
-
-    # initializer
-    # NOTE: This could be done with the "initializer" method instead
-    # http://guides.rubyonrails.org/generators.html#initializer
-    FileUtils.cp("#{@this_dir}/templates/config.rb", "#{@app_dir}/config/initializers/config.rb")
-
     # example config file
-    FileUtils.cp("#{@this_dir}/templates/config.yml", "#{@app_dir}/config/config.example.yml")
+    FileUtils.cp("#{@this_app}/lib/generators/templates/config.yml", "#{@new_app}/config/config.example.yml")
 
     # config file
-    # does not overwrite existing config file
-    if File.exist?("#{@app_dir}/config/config.yml")
-      return "Verify that your config/config.yml file matches the options in config/config.example.yml".cyan
-    else
-      FileUtils.cp("#{@this_dir}/templates/config.yml", "#{@app_dir}/config/config.yml")
-      return "Customize your app in config/config.yml".green
-    end
+    FileUtils.cp("#{@this_app}/lib/generators/templates/config.yml", "#{@new_app}/config/config.yml")
+    return "Customize your app in config/config.yml".green
+  end
+
+  def copy_initializer
+    # NOTE: This could be done with the "initializer" method instead
```

Comment:
I scribbled out what it would take:
```ruby
file = File.open("#{@this_app}/lib/generators/templates/config.rb", "rb")
initializer("config.rb", file.read)
file.close
```
I think the existing code is a much briefer, more straightforward approach 👍 
The method approach just seems quicker for dropping in static strings or new secrets etc.